format/spec.md

```diff
@@ -607,6 +607,7 @@ Notes:
 6. The following field ids are reserved on `data_file`: 141.
 7. `geometry`, this is a point: X = westernmost bound of all geometries in file, Y = northernmost bound of all geometries in file, Z is min value for all component points of all geometries in the file, M is min value of all component points of all geometries in the file. See Appendix D for encoding.
 8. `geometry`, this is a point: X = easternmost bound of all geometries in file, Y = southernmost bound of all geometries in file, Z is max value for all component points of all geometries in the file, M is max value of all component points of all geometries in the file. See Appendix D for encoding.
+9. `geometry`, the concepts of westernmost and easternmost values are explicitly introduced to address cases involving antimeridian crossing, where the `lower_bound` may be greater than `upper_bound`.
```

Comment:
Can this be part of 8?

Reply:
@szehon-ho It can but I feel since it mentions `westernmost` (7) and `easternmost` (8) so maybe it is worth to be a separate item? I mentioned in the spec: `see [7,9]` and `[8,9]`